angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -156,16 +145,20 @@ namespace :db_connector do
     ]
     add_indexes(indexes)
     conn = ActiveRecord::Base.connection
+    conn.execute 'TRUNCATE hpd_complaints;'
+    # ATTENTION: For demo purposes, we are only pulling limited addresses based on the hpd reg id. See the WHERE condition
     sql = "INSERT IGNORE INTO hpd_complaints
         (property_id, received_date, complaint_id, apartment, status, status_date, status_id,
-        complaint_type, major_category_id, minor_category_id, code_id)
+        complaint_type, major_category_id, major_category, minor_category_id, minor_category,
+        code_id, code, description)
         SELECT p.id, c.receiveddate, c.complaintid, c.apartment, c.status,
-        c.statusdate, c.statusid, prob.type, prob.majorcategoryid, prob.minorcategoryid,
-        prob.codeid
+        c.statusdate, c.statusid, prob.typeid, prob.majorcategory, prob.majorcategoryid, prob.minorcategory,
+        prob.minorcategoryid, prob.code, prob.codeid, prob.statusdescription
         FROM properties AS p INNER JOIN #{ENV['MYSQL_DATABASE_DATA']}.hpd_complaints AS c
             ON p.bbl = c.bbl
         INNER JOIN #{ENV['MYSQL_DATABASE_DATA']}.hpd_complaints_problems AS prob
-            ON c.complaintid = prob.complaintid;"
+            ON c.complaintid = prob.complaintid
+        WHERE p.hpd_registration_id IN(300980, 349726, 912547, 911741);"
```

Comment:
Checking this in to master makes me a little uncomfortable. Fine in the short term, but it'd be nice to separate this "WHERE" line out into its own commit and into a "Demo" branch.